site/content/docs/howto/cicd.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Using Pinniped for CI/CD cluster operations
+description: Using Pinniped for CI/CD cluster operations.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Use Pinniped for CI/CD
+    weight: 500
+    parent: howtos
+---
+
+This guide shows you how to configure Pinniped and your CI/CD system of choice to use Pinniped to administrate Kubernetes clusters.
+
+Note that the guide below assumes that you are using a non-human account within the IDP of your choice.
+It is never recommended to use a human's credentials in this way.
+
+## Prerequisites
+
+This how-to guide assumes that you have already configured the following Pinniped server-side components within your Kubernetes cluster(s):
+
+1. Pinniped Supervisor with a working FederationDomain and IdentityProvider (LDAP, AD, or OIDC)
+   1. Note that the Supervisor installation could be on a completely separate cluster unrelated to your CI/CD
```

Comment:
This would like nicer as a bullet point instead of an indented numbered list with only one item. Or maybe not indented at all, but just a second sentence in the first list item above. Whichever you prefer.

Reply:
Should have been a bullet point, see next push.